src/components/Dropdown/Dropdown.tsx

```diff
@@ -60,8 +60,18 @@ export interface DropdownProps {
    * @default uuidV4
    */
   id?: string;
+  /** Name used for input's form value. */
+  name?: string;
```

Comment:
Should we explicitly show it like this on JSDoc for other form elements also? 
Now some supports this `name` prop but it is not shown in the API doc on styleguidist.

Reply:
Good point, not sure but since dropdown does not extend upon other props like e.g. div, would feel a bit weird not to show it in the public API. Extended prop interfaces show the definition as first thing in IDEs when inspected. But I'm open for both alternatives.

Reply:
Yes, let's show the prop in public API. We need just to make the other places to follow same logic.